File: app/internal/class/infrastructure/repository/json/write/mapper/mapper.go
package mapper

import (
	"github.com/julianVelandia/EDteam/SOLIDyHexagonal/ProyectoCurso/internal/class/application/command"
	"github.com/julianVelandia/EDteam/SOLIDyHexagonal/ProyectoCurso/internal/class/infrastructure/repository/json/dto"
)

type Mapper struct{}

func (m Mapper) CommandToDTOClass(cmd command.Update) dto.ClassStudent {
	return dto.ClassStudent{
		ClassID: cmd.ClassID(),
		Title:   cmd.Title(),
	}
}

File: internal/classes/infrastructure/repository/json/write/mapper/mapper.py
from internal.classes.application.command.update import CommandUpdate
from internal.classes.infrastructure.repository.json.dto.classes import ClassesStudentDTO


class DTOClassesMapperWrite:
    def command_to_dto_class(self, cmd: CommandUpdate) -> ClassesStudentDTO:
        return ClassesStudentDTO(
            cmd.class_id(),
            cmd.title(),
        )
